css/templates/land-see/page.css

```diff
@@ -872,3 +875,111 @@ h4,
     margin-right: 0;
   }
 }
+
+/********** Story Detail Template **********/
+
+.land-see-story-detail .land-see-post-item-wide {
+  text-align: center;
+}
+
+.land-see-story-detail .land-see-hero-title {
+  font: 600 var(--h0)/var(--line-height-1) var(--font-primary);
```

Comment:
i see these definitions being written everywhere, can they be reused. 

As i told you before you could rewrite some typography classes and use them across - let me know what you think

Reply:
I think I understand what you're suggesting. Do you want me to override these:

```
.ampstart-title-lg {
  font-size: var(--h1);
  line-height: var(--line-height-6);
  letter-spacing: 0.06rem;
}

.ampstart-title-md {
  font-size: var(--h2);
  line-height: var(--line-height-5);
  letter-spacing: 0.06rem;
}

.ampstart-title-sm {
  font-size: var(--h3);
  line-height: var(--line-height-4);
  letter-spacing: 0.06rem;
}

...
```

And apply specific `font-family` and other attributes elsewhere?

My reasoning for keeping the one-line form of the `font` property was to reduce multiple lines for size/line-height.

Reply:
Assuming you have different styles of different Size text across your site , it makes sense to define them like this and reuse (easier for people to adapt), 

If you think you can fit your font definitions here  you can do that , it is up to you how you handle these classes

Reply:
I think that makes sense on this template, particularly, given that the story details are largely unique from the rest of the project. Should I be refactoring previous work?

Reply:
Yes - in another PR so that would be smaller and easier to find and replace.

Reply:
Ok. I'll take a stab at it in the next PR.